extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -1076,17 +1076,23 @@ export class Bind {
 
     switch (property) {
       case 'text':
-        element.textContent = String(newValue);
-        // If this is a <title> element in the <head>, update document title.
+        let updateTextContent = true;
+        const stringValue = String(newValue);
+
+        // textContent on <textarea> only works before interaction.
+        if (tag == 'TEXTAREA') {
```

Comment:
We use the strict '===' below `tag == 'TITLE'` but not here. Can we be consistent and use one over the other.